compact/optical_materials.xml

```diff
@@ -373,11 +408,11 @@
       1240*eV/210  10*mm
       1240*eV/200  0*mm
       "/>
-    <!-- BEGIN dRICH and pfRICH material optical properties
-      - to generate, follow: https://github.com/eic/drich-dev/blob/main/doc/material_tables.md
-      - original source for ATHENA/ECCE: https://github.com/cisbani/dRICh/blob/main/share/source/g4dRIChOptics.hh
-    -->
-    <!-- dRICH gas -->
+    <!-- BEGIN dRICh and pfRICh material properties
+         - dumped from fun4all implementation
+         - see https://github.com/cisbani/dRICh/blob/main/share/source/g4dRIChOptics.hh
+         -->
+    <!-- dRICh gas -->
     <matrix name="RINDEX__C2F6_DRICH" coldim="2" values="
```

Comment:
Careful here... this part is actually undoing #131, which is not something we want

Reply:
I check the change I made again and no undo here.
I re push the definition.xml and optical_material.xml, please help me check 4 files I commit.
Thanks!

Reply:
Much better, thanks!